.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: php
 dist: trusty
 php:
-  - '5.4'
-  - '5.5'
-  - '5.6'
   - '7.0'
   - '7.1'
 install: composer install
-script: phpunit --bootstrap="vendor/autoload.php" tests
\ No newline at end of file
+script: vendor/bin/phpunit
```

Comment:
Simplified this to ensure we're using the phpunit version specified via our composer.

Also, due to using composer and `phpunit.dist.xml`, it's no longer necessary to provide either the explicit bootstrap nor the tests directory